packages/preset-tagify/src/extractor.ts

```diff
@@ -0,0 +1,16 @@
+import type { Extractor } from '@unocss/core'
+
+export const MARKER = '__TAGIFY__'
+export const htmlTagRE = /<([\w\d-]+)/g
```

Comment:
👀
```suggestion
export const htmlTagRE = /<([\w\d-:]+)/g
```

Reply:
I wasn't sure if `:` is allowed as a legal HTML tag name according to [W3C standards](https://www.w3.org/TR/2011/WD-html5-20110525/syntax.html#:~:text=Tags%20contain%20a%20tag%20name,005A%20LATIN%20CAPITAL%20LETTER%20Z.), but it seems to work on chrome so why not :p